APPSEC-1087: S6413 Improve RSPEC (#3091)
SonarSource/rspec: rules/S6413/azureresourcemanager/rule.adoc

}
----

> This rule also applies to too short retention durations of logs for the following resources:

Reviewer: ```suggestion
This rule also applies to log retention periods that are too short, on the following resources:
```
